test/js/pageobjects/DocumentUpload.js

```diff
@@ -9,7 +20,91 @@ class DocumentUpload extends Base{
       return input
     })()}
 
-    getUploadButton() { return this.$('.onfido-sdk-ui-Uploader-button')}
+    upload(filename) {
+      const input = this.$('.onfido-sdk-ui-CustomFileInput-input')
+      const pathToTestFiles = '../../features/helpers/resources/'
+      const sendKeysToElement = input.sendKeys(path.join(__dirname, pathToTestFiles + filename))
+      return sendKeysToElement
+    }
+
+    /* eslint-disable no-undef */
+    copy(lang) { return locale(lang) }
+
+    verifyCrossDeviceUIElements() {
+      const documentUploadScreenCrossDeviceStrings = copy(lang).cross_device.switch_device
+      crossDeviceIcon.isDisplayed()
+      const crossDeviceHeaderText = crossDeviceHeader.getText()
+      expect(crossDeviceHeaderText).to.equal(documentUploadScreenCrossDeviceStrings.header)
+      crossDeviceHeader.isDisplayed()
+      const crossDeviceSubMessageText = crossDeviceSubMessage.getText()
+      expect(crossDeviceSubMessageText).to.equal(documentUploadScreenCrossDeviceStrings.header.submessage)
+      crossDeviceSubMessage.isDisplayed()
+      crossDeviceArrow.isDisplayed()
+    }
+
+    verifyUploaderIcon() {
+      uploaderIcon.isDisplayed()
+    }
+
+    verifyUploaderButton() {
+      const uploaderBtnText = uploaderBtn.getText()
+      expect(uploaderBtnText).to.equal(documentUploadLocale["capture"]["upload_file"])
```

Comment:
`documentUploadLocale` doesn't seem to be inscope in this method. is this method actually used?